charts/rasa-x/templates/db-migration-service.yaml

```diff
@@ -0,0 +1,21 @@
+{{ if eq "true" (include "db-migration-service.requiredVersion" .) }}
+{{ if .Values.separateDBMigrationService }}
+apiVersion: "v1"
+kind: "Service"
+metadata:
+  name: {{ include "rasa-x.fullname" . }}-db-migration-service-headless
```

Comment:
should we make the service name configurable?

Reply:
Hmm, It's hard for me to imagine a case that we'd need to change the service name, I'd leave it as it is 😄